test/draw/RasterRgbaTest.ooc

```diff
@@ -16,6 +16,15 @@ RasterRgbaTest: class extends Fixture {
 	sourceFlower := "test/draw/input/Flower.png"
 	init: func {
 		super("RasterRgbaTest")
+		this add("Test file not found", func {
+			try {
+				image := RasterRgba open("nonExistingFile")
+				expect(false)
+				expect(image, is equal to (null as RasterRgba))
+			} catch (e: Exception) {
+				expect(e message contains("Failed to load image: "))
```

Comment:
I'd rather avoid testing for the specific error message, I think it would be enough just to expect an exception.